.travis.yml

```diff
@@ -27,6 +27,8 @@ before_install:
 # NOTE: The latest version of dep requires go 1.13
 - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then mkdir -p /Users/travis/gopath/bin; fi
 - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then GO111MODULE=on scripts/install_and_setup.sh; fi
+- GO111MODULE=on go get sigs.k8s.io/controller-tools/cmd/controller-gen@v0.2.1
```

Comment:
we released 0.2.2 yesterday, may be use that. May be we should make these (kustomize below) version as some variables, so that it is easier to change them.